README.md

```diff
@@ -485,6 +485,51 @@ deleteAll() async {
 ```
 
 
+## Trouble-Shooting Section
+
+The *first* time I tried to change the path of the Flutter folder so that it could be accessed by the terminal without having to use several paths, it **did not work**, every time I closed the terminal Flutter was no longer recognized.</br> 
+
+![Screen Shot 2020-02-10 at 10 35 56](https://user-images.githubusercontent.com/27420533/74142666-41b7fd80-4bf1-11ea-9916-6b16261349aa.png)
```

Comment:
@miguelmartins17 all of these screenshots of terminal errors should be converted into code examples e.g:

```sh
command not found: flutter
```
This has _several_ benefits, the biggest is SEO (_people will be able to find this trouble shooting guide through Google_)

Reply:
I didn't see that one, sorry. 😄